packages/api/src/services/repository/mappers/repository-list.mapper.spec.ts

```diff
@@ -0,0 +1,49 @@
+import {RepositoryListMapper} from './repository-list.mapper';
+import {Repository, RepositoryList} from '../../../models/repositories';
+
+describe('RepositoryListMapper', () => {
+
+  let repositoryListMapper: RepositoryListMapper;
+
+  beforeEach(() => {
+    repositoryListMapper = new RepositoryListMapper();
+  });
+
+  test('Should return an instance of RepositoryList with repositories', () => {
+    const repoOneId = 'repo-one-id';
+    const repoTwoId = 'repo-two-id';
+    const repoThreeId = 'repo-three-id';
+    const repoThreeLocation = 'http://localhost:7002';
+    const rawRepositoryObjects = {
+      '': [{
+        id: repoOneId,
+        location: ''
+      }, {
+        id: repoTwoId,
+        location: ''
+      }],
+      'http://localhost:7002': [
+        {
+          id: repoThreeId,
+          location: repoThreeLocation
+        }
+      ]
+    };
+
+    const repositoryList = repositoryListMapper.mapToModel(rawRepositoryObjects);
+
+    expect(repositoryList).toBeInstanceOf(RepositoryList);
+    const repositories = repositoryList.getItems();
+    expect(repositories).toHaveLength(3);
+    expect(repositories).toContainEqual(new Repository({id: repoOneId, location: ''} as Repository));
```

Comment:
I didn't meant that, but rather mock with an actual response as you did above (without the location property for the local repositories) and verify the the Repository instance is being created without the location property.